app/controllers/api/v1/dishes_controller.rb

```diff
@@ -30,6 +30,20 @@ def desserts
 
         render json: dishes, status: 200
       end
+
+      def rate
+        dish = Dish.find(params[:id])
```

Comment:
This works fine, and I'm good with it but another alternative would be something like this

```
def rate
  dish = Dish.find params[:id]
  rating = dish.create_rating(rating_params)
  render json: rating
end
```

Might be a little more intention revealing